<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>361</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>161</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QRadioButton" name="radio_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio_1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio_2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio_3</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>input</string>
   </property>
   <widget class="QCheckBox" name="checkBox_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox 1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox 2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox 3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox 4</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>output</string>
   </property>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox 2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox 1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox 3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox 4</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
